Gruntfile.js

```diff
@@ -48,12 +48,9 @@ module.exports = function(grunt) {
   grunt.registerTask('lint', 'checks for syntax errors', function() {
     if (process.env.SKIP_LINT) {
       grunt.log.writeln('lint task skipped because of $SKIP_LINT');
-    } else if (semver.satisfies(process.version, '< 4.0.0')) {
```

Comment:
At the top of the file, you can now remove the line that requires the `semver` module. The lint command in the test suite caught this error for you :) Also, you can now remove `semver` from `package.json` because it's not used anywhere else.